backend/app.js

```diff
@@ -0,0 +1,30 @@
+import createError from 'http-errors';
+import express, { json, urlencoded } from 'express';
+import cookieParser from 'cookie-parser';
+import logger from 'morgan';
+
+const app = express();
+
+app.use(logger('dev'));
+app.use(json());
+app.use(urlencoded({ extended: false }));
+app.use(cookieParser());
+// app.use(static(join(__dirname, 'public')));
```

Comment:
comments?

Reply:
I changed the file after rebasing, So this commented code is gone.